conf/shiro.ini.template

```diff
@@ -66,8 +66,6 @@ user3 = password4, role2
 #knoxJwtRealm.cookieName = hadoop-jwt
 #knoxJwtRealm.publicKeyPath = /etc/zeppelin/conf/knox-sso.pem
 #
-#knoxJwtRealm.groupPrincipalMapping = group.principal.mapping
-#knoxJwtRealm.principalMapping = principal.mapping
```

Comment:
In addition to being unused, this default config throws an error, as these are expected to be strings of the format

`<user1>=<mappeduser1>;<user2>=<mappeduser2>`